web/components/MiniRoadmap.tsx

```diff
@@ -39,58 +41,76 @@ export const MiniRoadmap = (props: Props): ReactElement => {
     }
   };
 
-  const isLast = (stepNumber: number) => {
-    return roadmap?.steps[roadmap?.steps.length - 1].step_number === stepNumber;
+  const getSection = (sectionName: SectionType) => {
```

Comment:
maybe to be more careful with variable naming... this isn't the section name but rather the type, right?